### DISCUSSION TITLE
N00b Question i3-gaps config example

### DISCUSSION THREAD

**Discussion by threatseeker** [i3/i3#5435] [Q&A]
I really like the look that shows on the main i3-gaps GitHub.  Menu on top and square gaps.  Can I pull that config from somewhere, to look like that?

![687474703a2f2f692e696d6775722e636f6d2f7938735a45366f2e6a7067](https://user-images.githubusercontent.com/3854495/221443843-7a4c8830-a131-45b0-bd10-2bdd07db1c8c.jpg)

**Comment by Airblader** [comment]
I had all my configs here. However, this picture is quite (probably around 2015 or so), so the latest commit wouldn't exactly look like that anymore (i3bar should still be pretty much the same).

https://github.com/Airblader/dotfiles-manjaro
